docs/sources/auth/generic-oauth.md

```diff
@@ -38,10 +38,17 @@ token_url =
 api_url =
 allowed_domains = mycompany.com mycompany.org
 allow_sign_up = true
+tls_skip_verify_insecure = false
+tls_client_cert =
+tls_client_key =
+tls_client_ca =
 ```
 
 Set `api_url` to the resource that returns [OpenID UserInfo](https://connect2id.com/products/server/docs/api/userinfo) compatible information.
 
+You can also specify the SSL/TLS configuration used by the client. Set `tls_client_cert` to the path of the certificate, `tls_client_key` to the path containing the key and `tls_client_ca` to the path containing a trusted certificate authority list. 
+Finally, `tls_skip_verify_insecure` controls whether a client verifies the server's certificate chain and host name. If it is true, SSL/TLS accepts any certificate presented by the server and any host name in that certificate. In this mode, SSL/TLS is susceptible to man-in-the-middle attacks. This should be used only for testing.
```

Comment:
```suggestion
`tls_skip_verify_insecure` controls whether a client verifies the server's certificate chain and host name. If it is true, then SSL/TLS accepts any certificate presented by the server and any host name in that certificate. _You should only use this for testing_, because this mode leaves SSL/TLS susceptible to man-in-the-middle attacks.
```